tests/PluralRulesTest.elm

```diff
@@ -54,6 +55,38 @@ fromFloatInt =
         ]
 
 
+rulesWithKind : Rules
+rulesWithKind =
```

Comment:
I wonder if sometime we'll need to somehow prefix these with the language they're for :joy: Today is not that day, but still...